source/form.html

```diff
@@ -0,0 +1,243 @@
+        <span class="visually-hidden">Логотип магазина «Кэт Энерджи»</span>
+      </a>
+      <nav class="lead-navigation">
+        <button class="lead-navigation__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="lead-navigation__wrapper">
+          <ul class="lead-navigation__list site-list">
+            <li class="site-list__item ">
+              <a href="index.html">
+                Главная
+              </a>
+            </li>
+            <li class="site-list__item">
+              <a href="catalog.html">
+                Каталог продукции
+              </a>
+            </li>
+            <li class="site-list__item site-list__item--active">
+              <a>
+                Подбор программы
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main-container main">
+      <h1 class="main__title">Подбор программы</h1>
+      <p class="main__description">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+      <form class="main__form site-form" action="https://echo.htmlacademy.ru/" method="post">
+        <p class="site-form__group">
+          <label class="site-form__label" for="field-name">ИМЯ:*</label>
+          <input
+            class="site-form__field"
+            placeholder="Барсик"
+            type="text"
+            name="field-name"
+            id="field-name"
+            required
+          >
+        </p>
+        <p class="site-form__group">
+          <label class="site-form__label" for="field-weight">Вес (кг):*</label>
+          <input
+            class="site-form__field"
+            placeholder="7"
+            type="text"
+            name="field-weight"
+            id="field-weight"
+            required
+          >
+        </p>
+        <p class="site-form__group">
+          <label class="site-form__label" for="field-age">Возраст (лет):</label>
+          <input
+            class="site-form__field"
+            placeholder="7"
+            type="text"
+            name="field-age"
+            id="field-age"
+          >
+        </p>
+        <fieldset class="site-form__solution">
+          <ul class="solution__list">
+            <li class="solution__item">
+              <label class="solution__label">
+                <input
+                  class="solution__input site-input"
+                  type="radio"
+                  name="solution-group"
+                  value="slim"
+                  checked
+                >
+                <span class="site-input__caption"></span>
+                похудение
```

Comment:
Нужно текст писать как в обычном предложении
```suggestion
                Похудение
```